basyx/aas/model/base.py

```diff
@@ -1468,7 +1468,10 @@ def __init__(self,
         self.value_type: Optional[DataTypeDefXsd] = value_type
         self._value: Optional[ValueDataType]
         self.value = value
-        self.refers_to: Iterable[ModelReference] = refers_to
+        if refers_to is None:
+            self.refers_to: List[ModelReference] = []
+        else:
+            self.refers_to = refers_to
```

Comment:
```suggestion
        self.refers_to: List[ModelReference] = refers_to if refers_to is not None else []
```

Reply:
I think I would keep the parameter of `__init__()` as it was before, accepting any `Iterable[ModelReference]`. The attribute could still be a `List` or a `Set`, just initializing it as `list(refers_to)` or `set(refers_to)`.
I think it would make sense to use a `Set` here, as References are hashable and having duplicate references probably wouldn't make much sense.

Reply:
Fixed in daa770f